crates/burn-tensor/src/tensor/api/base.rs

```diff
@@ -213,7 +215,13 @@ where
         Tensor::new(K::reshape(self.primitive, shape))
     }
 
-    /// Transpose the tensor.
+    /// Transpose the tensor. Permute the 2 last dimensions of the tensor.
```

Comment:
Forgot this edit in my previous review comment. That's on me 😅 

```suggestion
    /// Transpose the tensor.
```